### ISSUE TITLE
Generate bash completions from cobra

### ISSUE THREAD

**Issue by dlorenc** [kubernetes/minikube#172]
https://github.com/spf13/cobra#generating-bash-completions-for-your-command

We'll need installer scripts before we can add this, though. Ref #12

**Comment by jimmidyson** [comment]
And zsh please.

**Comment by aaron-prindle** [comment]
It looks like cobra does not have zsh auto-completion built in: https://github.com/spf13/cobra/issues/107.  Closing this issue as it referenced bash, please re-open another issue for zsh.
